src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperPartialExtensions.cs

```diff
@@ -241,6 +241,94 @@ public static class HtmlHelperPartialExtensions
         /// <param name="partialViewName">
         /// The name of the partial view used to create the HTML markup. Must not be <c>null</c>.
```

Comment:
The comments are a little weird but don't seem to do much harm. I wouldn't go out of the way to either add these types of comments nor remove them; if they're there, they can stay.